packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -70,7 +70,7 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   );
 
   return (
-    <Stack className={chatContainer} grow>
+    <Stack id="ChatComposite" className={chatContainer} grow>
```

Comment:
Is it possible a user may have two chat composites in their app? (should this be a classname instead?)

Reply:
Good Point.